src/internal/connector/exchange/api/api.go

```diff
@@ -87,6 +87,18 @@ func newService(creds account.M365Config) (*graph.Service, error) {
 	return graph.NewService(adapter), nil
 }
 
+func (c Client) Contacts() Contacts {
+	return Contacts{c}
+}
+
+func (c Client) Events() Events {
+	return Events{c}
+}
+
+func (c Client) Mail() Mail {
+	return Mail{c}
+}
+
```

Comment:
This appears a little circular. The creds and service associated are available within the client already. As this is being merged into main. A small comment about about the purpose of these calls would be helpful It's also unclear why there wouldn't be similar calls for onedrive or sharepoint.

Reply:
The comment would be, succinctly, `interface compliance wrappers`.  Not much more to tell than that.

> It's also unclear why there wouldn't be similar calls for onedrive or sharepoint.

Note the package location.  onedrive and sharepoint have no api packages as of yet.